packages/riipen-ui/src/components/TableRow.jsx

```diff
@@ -0,0 +1,78 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+
+import ThemeContext from "../styles/ThemeContext";
+
+class TableRow extends React.Component {
+  static propTypes = {
+    /**
+     * Children to render in the tr element
+     */
+    children: PropTypes.node,
+
+    /**
+     * Classes to apply to the root element
+     */
+    classes: PropTypes.arrayOf(PropTypes.string),
+
+    /**
+     * The number of columns the row should span
+     */
+    span: PropTypes.number,
+
+    /**
+     * Whether to highlight the row on hover
+     */
+    hover: PropTypes.bool,
+
+    /**
+     * Whether to render the border for the row
+     */
+    border: PropTypes.bool
+  };
+
+  static defaultProps = {
+    span: 1,
+    border: true
+  };
+
+  static contextType = ThemeContext;
+
+  render() {
+    const { children, classes, span, hover, border } = this.props;
+
+    const theme = this.context;
+
+    return (
+      <React.Fragment>
+        <tr className={clsx(classes)} rowSpan={span}>
+          {children}
+        </tr>
+        <style jsx>{`
+          tr {
+            ${border
+              ? `background: linear-gradient(
+                to bottom,
+                transparent calc(100% - 1px),
+                ${theme.palette.grey[400]} 0%,
+                ${theme.palette.grey[400]}
+              )
+              bottom / calc(100% - ${theme.spacing(8)}px);`
+              : ""}
+            border-left: 3px solid transparent;
```

Comment:
I am confused. What is this doing exactly?

Reply:
This is how you have to render lines between rows in css. you can't apply border-bottom to a tr element, so instead you have to create a background that looks like a border. This one is taken directly from the Table component in the web repo

Reply:
weirdness, it seems like if you add `border-collapse: collapse` to the table you can directly apply borders to `tr` elements. I tested it out in a sandbox and seems to work. Might be worth testing out to clean this up?

Reply:
thanks, ill take a look